includes/Modules/Site_Verification.php

```diff
@@ -499,4 +504,16 @@ private function serve_verification_file( $verification_token ) {
 
 		// If the user does not have the necessary permissions then let the request pass through.
 	}
+
+
+	/**
+	 * Returns TRUE to indicate that this module should be always active.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool Returns `true` indicating that this module should be activated all the time.
+	 */
+	public static function is_force_active() {
+		return true;
+	}
```

Comment:
Same as above, now that this is here, we should probably remove the duplicate directive in `Site_Verification::setup_info`.